Makefile

```diff
@@ -82,6 +82,43 @@ docs:
 servedocs: docs
 	watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
+website-api:
+	rm -f website/_api/sceptre.rst
+	rm -f website/_api/modules.rst
+	sphinx-apidoc -o website/_api sceptre
+	$(MAKE) -C website/_api clean
+	$(MAKE) -C website/_api html
+	mkdir -p website/docs/api
+	rm -rf website/docs/api/_static
+	mkdir -p website/docs/api/_static/
+	cp -r website/_api/_build/html/_static website/docs/api/
+	rm -f website/docs/api/sceptre.html
+	cp website/_api/_build/html/sceptre.html website/docs/api/sceptre.html
```

Comment:
Why only this file? Doesn't the sphinx api pages need sceptre.resolvers.html, sceptre.hooks.html etc..?